sklearn/ensemble/tests/test_gradient_boosting.py

```diff
@@ -82,6 +82,12 @@ def test_classification_toy():
 
 def test_classifier_parameter_checks():
     # Check input parameter validation for GradientBoostingClassifier.
+    assert_raise_message(ValueError, "Presort must be a boolean",
+                         GradientBoostingClassifier(presort='invalid')
+                         .fit, X, y)
+    assert_raise_message(ValueError, "Presort must be a boolean",
```

Comment:
At a glance, the second test seems duplicate with the first one? It might not be good for such a minor thing.

Reply:
Oh I get it!